packages/spectacle/src/hooks/use-broadcast-channel.ts

```diff
@@ -5,10 +5,11 @@ import { DeckView } from './use-deck-state';
 
 const noop = () => {};
 let safeWindow: any = {};
-if (typeof window !== "undefined") {
+if (typeof window !== 'undefined') {
   safeWindow = window;
 }
-const BroadcastChannel = safeWindow.BroadcastChannel || BroadcastChannelPolyfill;
+const BroadcastChannel =
+  safeWindow.BroadcastChannel || BroadcastChannelPolyfill;
```

Comment:
This was merged in from a community PR and escaped formatting. Just a quick fix to get CI passing.